### PR TITLE
js-fetch-exercise lilas-saleh maher-suleyman

### PR DESCRIPTION
**Pull request by lilas-saleh**

### CODE REVIEW

File: madlibs-project/lilas-tugba/madlibs.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ */
+// PARSING !! 
+function parseStory(rawStory) {
+    let array = [];
+    let newStory = rawStory;
+    let splitting = newStory.split(" "); // splitting the text
+    //console.log(splitting)
+    let noun = /n(?=])/ ;
+    let verb = /v(?=])/ ;
+    let adjective =/a(?=])/;
```
Comment (louisrli) [inline madlibs-project/lilas-tugba/madlibs.js:37]: wow this is looking good! if it works :)

File: madlibs-project/lilas-tugba/madlibs.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ *  { word: "to", },
+ *  { word: "the", },
+ *  { word: "store", pos: "noun" }
+ *  { word: "," }
+ *  ....
+ * 
+ * There are multiple ways to do this, but you may want to use regular expressions.
+ * Please go through this lesson: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/regular-expressions/
+ */
+// PARSING !!
```
Comment (louisrli) [inline madlibs-project/lilas-tugba/madlibs.js:29]: are you that excited?

**Comment by louisrli** [comment]
can you try separating the js-fetch-exercise from mad libs? hint: use `git checkout -b`

**Comment by lilas-saleh** [comment]
@louisrli I checked out 2 branches then check out master and fetch all. 
So far, is this correct?

**Comment by louisrli** [comment]
hmmm...not quite.

you'll want to do this:

```
git checkout -b withoutmadlibs
<delete teh mad libs folder>
<commit>
git push origin withoutmadlibs
git checkout master
```
then on github you can create a new pull request from the withoutmadlibs branch

**Comment by lilas-saleh** [comment]
@louisrli while I am in bootcamp-istanbul-2020mar I did the following: 

`git checkout -b withoutmadlibs`
`rm -rf madlibs-project`
`git add .`
`git commit -m "delets mad libs folder"`
`git push origin withoutmadlibs`
`git checkout master`

**Pull request closed without merging**